.doc_gen/metadata/sns_metadata.yaml

```diff
@@ -5,6 +5,13 @@ sns_GetTopicAttributes:
   synopsis: get the properties of an &SNS; topic.
   category:
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/sns
+          excerpts:
+            - snippet_tags:
+                - sns.cs.get_topic_attributes.main
```

Comment:
The backend validator is rejecting this. It looks like your tag should be:
```
SNS.dotnetv3.GetTopicAttributesExample
```

Reply:
fixed